CHANGELOG.md

```diff
@@ -36,6 +36,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- **CUMULUS-2123**
+  - Added `cumulus-rds-tf` DB cluster module to `tf-modules` that adds a severless RDS Aurora/ PostgreSQL database cluster to meet the PostgreSQL requirements for the 2.1.x release series
```

Comment:
Am I missing something? I don't see this added.

Reply:
It was in the original (merged) PR - this is just properly updating the CHANGELOG notes for the ticket.